code/modules/power/singularity/emitter.dm

```diff
@@ -199,8 +199,6 @@
 
 /obj/machinery/power/emitter/proc/fire_beam(atom/targeted_atom, mob/user)
 	var/turf/targets_from = get_turf(src)
-	if(targeted_atom == user|| targeted_atom == targets_from)

```

Comment:
Apparently my commit wasn't recognizing as changing this line but uh... yea it's been moved now.